src/main/webapp/app/shared/user-settings/ide-preferences/ide-settings.component.ts

```diff
@@ -10,30 +11,32 @@ import { IdeSettingsService } from 'app/shared/user-settings/ide-preferences/ide
     styleUrls: ['./ide-settings.scss'],
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class IdeSettingsComponent implements OnInit {
+export class IdeSettingsComponent implements OnInit, OnDestroy {
     protected readonly ProgrammingLanguage = ProgrammingLanguage;
     protected readonly faPlus = faPlus;
     protected readonly faTrash = faTrash;
-    PREDEFINED_IDE: Ide[] = [{ name: 'VS Code', deepLink: 'vscode://vscode.git/clone?url={cloneUrl}' }];
+    PREDEFINED_IDE: Ide[];
```

Comment:
I think we shouldn't use SCREAMING_SNAKE_CASE here, as it's not a constant.
 https://docs.artemis.cit.tum.de/dev/guidelines/server.html#naming-convention